debian/rules

```diff
@@ -15,7 +15,7 @@ else ifeq ($(shell dpkg-vendor --is endless && echo yes),yes)
 	dbx=debian/endless-uefi-dbx.esl
 	distributor=endless
 	oslabel="Endless OS"
-COMMON_OPTIONS ?= VENDOR_DBX_FILE=$(dbx) OSLABEL=$(oslabel) FALLBACK_VERBOSE_WAIT=3000000
+COMMON_OPTIONS ?= VENDOR_DBX_FILE=$(dbx) OSLABEL=$(oslabel) FALLBACK_VERBOSE_WAIT=3000000 FALLBACK_NEVER_REBOOT=1
```

Comment:
We should add some line continuations here at some point...

Reply:
Yes, let's keep that in mind for the future. That said, hopefully the future pans out in a way that we'll actually drop this flag :-)